tables/th-g0-backward.utb

```diff
@@ -0,0 +1,222 @@
+# liblouis: Thai Current Braille System.
+#
+#-index-name: Thai
+#-display-name: Thai current braille
```

Comment:
The display name of the other table is "Thai braille". Why is this one called "Thai current braille" then? Shouldn't it be the same?

Reply:
You can change to Thai braille.

Reply:
Never mind what I said earlier. The display names should be distinct because they implement different braille codes (6 vs. 8 dot).